### ISSUE TITLE
interactive git rebasing causes ConEmu to crash

### ISSUE THREAD

**Issue by ConEmu** [ConEmu/old-issues#1060]
_From [kevin.ca...@bromium.com](https://code.google.com/u/101475368719670331332/) on May 09, 2013 15:46:46_

Required information! OS version: Win7 sp1 x64 ConEmu version: ConEmuSetup.130427.exe
Far version (if you are using Far Manager): not knowingly.

Sometimes when I do interactive git rebases from within cmd.exe in ConEmu , ConEmuC64.exe crashes.  (I have git installed from Git-1.8.1.2-preview20130201.exe.)

```
git rebase -i my_branch~3
```

To recover, I have to kill vim.exe, since the ConEmu tab it was in is crashed.

For example, see the attached WinDbg log.

**Attachment:** [conemu-crash.windbg.txt](http://code.google.com/p/conemu-maximus5/issues/detail?id=1061)

_Original issue: http://code.google.com/p/conemu-maximus5/issues/detail?id=1061_

**Comment by ConEmu** [comment]
_From [ConEmu.M...@gmail.com](https://code.google.com/u/116287257743594353797/) on May 10, 2013 03:39:18_

I'm interested in full memory dump of crashed process. Log is not enough.

**Comment by ConEmu** [comment]
_From [boxmon...@gmail.com](https://code.google.com/u/102446199106730395162/) on May 11, 2013 19:11:48_

Vim in git bash in conemu sometimes crashes for me too. What's the best way to get a usable crash dump?

**Comment by ConEmu** [comment]
_From [l...@bowbak.org](https://code.google.com/u/100252683791590041677/) on June 12, 2013 07:29:10_

This is the same as issue `#1060` Just FYI, I switched to Windows 8 and ConEmu 130604 and this issue appears to be working better. I can now interactive rebase using vim.exe as the editor without it crashing most of the time. However, in the console window I just did a rebase in I started up vim by itself and the window crashed.
